src/modules/autocomplete.js

```diff
@@ -137,7 +142,12 @@ class Autocomplete {
       headers['User-Agent'] = userParameters.userAgent;
     }
 
-    return fetch(requestUrl, { headers }).then((response) => {
+    // Handle timeout if specified
+    if (networkParameters.timeout && typeof networkParameters.timeout === 'number') {
```

Comment:
Should we do some destructuring here for `timeout`?

Reply:
We could - I don't really think it adds anything but extra lines of code?

Reply:
I think it's fine to leave it as is for now until we add other options to the networkParameters? up to you 😄

Reply:
👍  agreed! thanks for the review!